### ISSUE TITLE
use sequential pool splits to increase efficiency even further

### ISSUE THREAD

**Issue by phiweger** [phiweger/clonepool#22]
https://www.medrxiv.org/content/10.1101/2020.04.10.20061176v1

**Comment by phiweger** [comment]
We only need to simulate some more: the unresolved samples will have many more positive samples among them, so replicates should make even more sense ... on the other hand, the paper above does not redistribute samples, which we would have to do given our current setup.

maybe we should be practical and add a fn:

```bash
clonepool split data/result.csv data/split/
```

which then contains all the positive pools split in two. then again

```bash
for i in data/split/ do;
    clonepool resolve $i ...
done
```

For now I'll simply link this paper in the draft.

**Comment by xileF1337** [comment]
Well, I haven't all of the paper yet, but if I understand correctly they simply estimate the new prevalence and adapt their scheme in the next pooling step. We, however, argued that in a second step all samples should be tested individually because otherwise it may take unbearably long to get the results for an individual patient (if a third, fourth etc step are required). 

Anyway, we have to compare our method to theirs ...

**Comment by phiweger** [comment]
yeah, exactly the point. if you'd split the pool in iteration 2, you're likely to still get unresolved samples, leading to at least 3 PCRs which I think is an unacceptable delay at least in our clinic.
